### DISCUSSION TITLE
Will there be support for the expansion packs?

### DISCUSSION THREAD

**Discussion by saulg0odman** [sultim-t/xash-rt#102] [General]
I'm wondering if support for opposing force and blue shift will be available?

**Comment by sultim-t** [comment]
It's possible, requires recompiling their .dll-s to x64, and tweaking all of the maps for RT (it won't be me 🍞)

**Reply by RadioactiveE3** [reply]
Understandable, however it would be quite fantastic if you could support expansions at some point in the future. Awesome work by the way.
